package/upgrade/upgrade_manifests.sh

```diff
@@ -1367,6 +1369,23 @@ rules:
 EOF
 }
 
+fleet_agent_timestamp(){
+  wait_for_deployment cattle-fleet-local-system fleet-agent &> /dev/null
+  time=$(kubectl get deploy -n cattle-fleet-local-system fleet-agent -o json | jq -r .metadata.creationTimestamp)
```

Comment:
`kubectl get deploy` may fail because the above `wait_for_deployment` just success on the old deployment